server/server.go

```diff
@@ -98,8 +98,8 @@ type Server struct {
 	listener         net.Listener
 	gacc             *Account
 	sys              *internal
-	accounts         map[string]*Account
-	activeAccounts   int
+	accounts         sync.Map
+	activeAccounts   int32
```

Comment:
Looks like issue https://github.com/golang/go/issues/599 is still opened, so I usually put the fields I want to use atomic ops on at top of structure, and often use 64bits versions. Not sure if this is an issue or not (you would have to try with the linux 386 exe to see if it works or not.

Reply:
Meaning it will fail or just be slow?

Reply:
No perf diff on OSX. Will check linux.

Reply:
I think very early reports were that it would core dump on 386.

Reply:
None in linux either.

Reply:
Ah, not sure how to test that atm..

Reply:
Thx!